README.md

```diff
@@ -1,75 +1,55 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
```

Comment:
Please try to add the description of your project. It will help all those people who will come in future to know what you have done in this project.